repeater.go

```diff
@@ -96,14 +96,14 @@ func (rep *repeater) terminate() {
 	terminate(rep.cmd)
 }
 
-func getWinSshVersion() string {
+func getWinSshVersion(path string) string {
```

Comment:
Maybe this function is no longer used? Actually it is used by two tests, but it is ok to remove the tests.